bellboy/utils/messages.py

```diff
@@ -96,3 +96,36 @@ def __init__(self, eventType, eventData):
 
     def __str__(self):
         return self.eventType.name
+
+
+"""LCD messages"""
+
+
+class LCDReq(Enum):
+    SETUP, DISPLAY= range(2)
+
+
+class LCDResp(Enum):
+    SET, DISPLAYING= range(2)
+
+
+class LCDMsg:
+    def __init__(self, msgType, LCDNumber=None):
```

Comment:
add parameters ``displayText (string)`` and ``displayDuration (int/long)`` for messages that will need those